src/Microsoft.OData.Core/ODataSimplifiedOptions.cs

```diff
@@ -14,20 +14,33 @@ namespace Microsoft.OData
     public sealed class ODataSimplifiedOptions
     {
         /// <summary>
-        /// Instance for <see cref="ODataSimplifiedOptions"/>.
+        /// Constructor of ODataSimplifiedOptions
         /// </summary>
-        private static readonly ODataSimplifiedOptions DefaultOptions = new ODataSimplifiedOptions();
+        public ODataSimplifiedOptions() : this(null /*version*/)
+        {
+        }
 
         /// <summary>
         /// Constructor of ODataSimplifiedOptions
         /// </summary>
-        public ODataSimplifiedOptions()
+        /// <param name="version">The ODataVersion to create Default Options for.</param>
+        public ODataSimplifiedOptions(ODataVersion? version)

```

Comment:
>public [](start = 8, length = 6)

Can this be internal/private instead of public, since default ctor and GetODataSimplifiedOptions() already provide public API?

Reply:
GetODataSimplifiedOptions is internal.  This is the only public interface to create an instance of a version-specific ODataSimplifiedOptions for 4.01.

---
In reply to: [150166363](https://github.com/OData/odata.net/pull/989#discussion_r150166363) [](ancestors = 150166363)